static/app/components/events/interfaces/crashContent/stackTrace/content.tsx

```diff
@@ -172,240 +180,188 @@ class Content extends Component<Props, State> {
-            isExpanded={expandFirstFrame && lastFrameIdx === frameIdx}
-            emptySourceNotation={lastFrameIdx === frameIdx && frameIdx === 0}
-            isOnlyFrame={(data.frames ?? []).length === 1}
-            nextFrame={nextFrame}
-            prevFrame={prevFrame}
-            platform={platform}
-            timesRepeated={nRepeats}
-            showingAbsoluteAddress={showingAbsoluteAddresses}
-            onAddressToggle={this.handleToggleAddresses}
-            image={image}
-            maxLengthOfRelativeAddress={maxLengthOfAllRelativeAddresses}
-            registers={{}} // TODO: Fix registers
-            isFrameAfterLastNonApp={isFrameAfterLastNonApp}
-            includeSystemFrames={includeSystemFrames}
-            onFunctionNameToggle={this.handleToggleFunctionName}
-            onShowFramesToggle={e => {
-              this.handleToggleFrames(e, frameIdx);
-            }}
-            isSubFrame={hiddenFrameIndices.includes(frameIdx)}
-            isShowFramesToggleExpanded={toggleFrameMap[frameIdx]}
-            showCompleteFunctionName={showCompleteFunctionName}
-            isHoverPreviewed={isHoverPreviewed}
-            frameMeta={meta?.frames?.[frameIdx]}
-            registersMeta={meta?.registers}
-            debugFrames={debugFrames}
-            isANR={isANR}
-            threadId={threadId}
-            lockAddress={lockAddress}
-            hiddenFrameCount={frameCountMap[frameIdx]}
-          />
-        );
+        const frameProps = {
+          event,
+          data: frame,
+          isExpanded: expandFirstFrame && lastFrameIndex === frameIndex,
+          emptySourceNotation: lastFrameIndex === frameIndex && frameIndex === 0,
+          isOnlyFrame: (data.frames ?? []).length === 1,
+          nextFrame,
+          prevFrame,
+          platform,
+          timesRepeated: nRepeats,
+          showingAbsoluteAddress: showingAbsoluteAddresses,
+          onAddressToggle: handleToggleAddresses,
+          image: findImageForAddress(frame.instructionAddr, frame.addrMode),
+          maxLengthOfRelativeAddress: maxLengthOfAllRelativeAddresses,
+          registers: {}, // TODO: Fix registers
+          isFrameAfterLastNonApp: isFrameAfterLastNonApp(),
+          includeSystemFrames,
+          onFunctionNameToggle: handleToggleFunctionName,
+          onShowFramesToggle: (e: React.MouseEvent<HTMLElement>) => {
+            handleToggleFrames(e, frameIndex);
+          },
+          isSubFrame: hiddenFrameIndices.includes(frameIndex),
+          isShowFramesToggleExpanded: toggleFrameMap[frameIndex],
+          showCompleteFunctionName,
+          isHoverPreviewed,
+          frameMeta: meta?.frames?.[frameIndex],
+          registersMeta: meta?.registers,
+          debugFrames,
+          isANR,
+          threadId,
+          lockAddress,
+          hiddenFrameCount: frameCountMap[frameIndex],
+        };
+
+        nRepeats = 0;
+
+        if (frameIndex === firstFrameOmitted) {
```

Comment:
Just want to double check that this works the same as it did before? Since this logic is inside the if statement, but before it was outside of it.